public/scripts/socket-dashboard.js

```diff
@@ -1,46 +1,70 @@
 /**
  * Search for repositories
  */
-var search = function () {
-  var username = $("#orgs").val().split(":")[1];
-  const searchBarInput = $("#search_bar");
-  const searchResultDiv = $("#search_result");
+var subProjectId = 0; // sub projects dynamic id
+var repositorySelectIds = []; // array for storing repositories select element
+
+var registerDisableChecker = function () {
+  var flag = true; // flag to check wheather any of the select is
```

Comment:
typo: _wheather_ -> _whether_

.. any of the select is ..? This sentence looks incomplete.